docs/api.asciidoc

```diff
@@ -349,30 +323,8 @@ does not contain a span.
 [[span-parentid]]
 ==== `func (*Span) ParentID() SpanID`
 
-ParentID returns the span's Parent ID generating a parent span ID if one has
-not already been set and tx has not been ended.
-If tx is nil or has been ended, a zero (invalid) SpanID is returned.
-
-[source,go]
-----
-// TraceContext returns a map[string]interface{} containing the trace
-// context of the transaction and span contained in ctx, if any.
-func TraceContext(ctx context.Context) map[string]interface{} {
-	tx := apm.TransactionFromContext(ctx)
-	if tx == nil {
-		return nil
-	}
-	traceContext := tx.TraceContext()
-	fields := map[string]interface{}{
-		"trace_id":       traceContext.Trace,
-		"transaction_id": traceContext.Span,
-	}
-	if span := apm.SpanFromContext(ctx); span != nil {
-		fields["span_id"] = span.ParentID()
-	}
-	return fields
-}
-----
+ParentID returns the ID of the span's parent span or transaction.
+If Span is nil or has been ended, a zero (invalid) SpanID is returned.
```

Comment:
```suggestion
ParentID returns the ID of the span's parent.
```

Spans can't exist without a parent. It's true that a zero SpanID is returned for a nil Span, but I'm not sure we need to explain that in these docs?